node_cli/utils/docker_utils.py

```diff
@@ -127,6 +127,27 @@ def get_logs_backup_filepath(container: Container) -> str:
     return os.path.join(REMOVED_CONTAINERS_FOLDER_PATH, log_file_name)
 
 
+def init_volume(name: str, size: int):
+    if is_volume_exists(name):
```

Comment:
Two docker client objects will be created here instead of one. I think it will be better to allow specific docker client and set default value.